src/api/objects/ObjectAPI.js

```diff
@@ -165,7 +165,11 @@ define([
             throw new Error('Provider does not support get!');
         }
 
-        return provider.get(identifier);
+        if (fallback) {
+            return this.fallbackProvider.get(identifier);
```

Comment:
The fallbackProvider.get call here invokes this:
https://github.com/nasa/openmct/blob/master/src/adapter/services/LegacyObjectAPIInterceptor.js#L123